packages/utils/src/cda-converter/cda-conversion-testing-script.ts

```diff
@@ -0,0 +1,206 @@
+// The objective of these tests are to test two things
+// 1. That we generate valid CDA that our CDA->FHIR converter doesn't error on. We cant know if EPIC won't error on our CDA yet, but not erroring on our own CDA is a good start
+// 2. That our FHIR to CDA converter generates accurate CDA, such that when our CDA is converted back to FHIR, we get the same FHIR bundle as the original FHIR bundle. Obviously this introduces
+//    a dependency on another converter, but for now its the easiest approach since short of manually having exact string comparisons between two CDAs, we can't know if our CDA is accurate.
+
+import fs from "fs";
+import path from "path";
+import { splitBundleByCompositions } from "@metriport/core/fhir-to-cda/composition-splitter";
+import { generateCdaFromFhirBundle } from "@metriport/core/fhir-to-cda/cda-generators";
+import axios, { AxiosInstance } from "axios";
+import { Bundle } from "@medplum/fhirtypes";
+
+const fhirBaseUrl = "http://localhost:8888";
+const baseInputFolder = "./src/cda-converter/scratch/";
+
+if (!fs.existsSync(baseInputFolder)) {
+  console.error("Base input folder does not exist:", baseInputFolder);
+  process.exit(1);
+}
+
+fs.readdir(baseInputFolder, (err, files) => {
```

Comment:
As a good programming practice and to keep our pattern, create a `main()` function and call it at the end of the file, please.